src/inspect.js

```diff
@@ -47,11 +48,8 @@ export function silenceError (p) {
 	p._runAction(silencer)
 }
 
-const silencer = {
-	fulfilled () {},
-	rejected: setHandled
-}
-
-function setHandled (rejected) {
-	rejected._state |= HANDLED
+const silencer = new Action(null)
```

Comment:
I think this is the only thing.  Since there's no compelling reason to change it, I'd prefer to keep silencer as it is in master.